tests/integration/test_repository.py

```diff
@@ -788,16 +801,32 @@ async def test_repo_list_and_count_method_empty(book_repo: BookRepository) -> No
     assert len(collection) == 0
 
 
+@pytest.fixture()
+def frozen_datetime() -> Generator[Coordinates, None, None]:
+    with travel(datetime.utcnow, tick=False) as frozen:
+        yield frozen
+
+
 async def test_repo_created_updated(
+    frozen_datetime: Coordinates,
     author_repo: AuthorRepository,
     book_model: type[AnyBook],
     repository_pk_type: RepositoryPKType,
 ) -> None:
     author = await maybe_async(author_repo.get_one(name="Agatha Christie"))
+
+    if isinstance(author_repo.session, AsyncSession):
+        _ = SQLAlchemyAsyncConfig(
```

Comment:
Why are these created but never used?

Reply:
Because we need to invoke the `__post_init__` methods to import the event listener when it's configured.